src/pages/check-email/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+import SEO from '../../components/containers/seo'
+import { IconGrid } from './_icon-grid'
+import Button from 'components/form/button'
+import device from 'themes/device'
+import Layout from 'components/layout/layout'
+import Modal, { useModal } from 'components/elements/modal'
+import SignupModal from 'components/elements/signup-modal'
+import { SectionContainer } from 'components/containers/container'
+import { localize, WithIntl } from 'components/localization'
+
+const SectionHeader = styled.p`
+    font-size: 2rem;
+    color: var(--color-black-2);
+    text-align: left;
+`
+const Container = styled.div`
```

Comment:
is this container different from the container from component?

Reply:
yes, it is. Actually, I added `GridContainer` to Container component.

Reply:
thats great, thank you 👍